Hank-iOS/AnthropicService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AnthropicService.swift
+//  Hank-iOS
+//
+//  Created by Craig Atallah Frost on 26/06/2024.
+//
+
+import Foundation
+
+class AnthropicService {
+    private let apiKey = Env.ANTHROPIC_API_KEY
+    private let urlString = "https://api.anthropic.com/v1/messages"
+    
+    func sendMessages(_ messages: [Message]) async throws -> String {
```

Comment:
From the name `sendMessages`, I initially imagined that this method would be responsible only for sending messages. However, since it also includes receiving and returning responses, I thought it might be better to use a more explicit name.

Reply:
Ah, good note! Thanks 🙂